ReleaseProcessScript/tools/Core/git_base_functions.ps1

```diff
@@ -22,14 +22,36 @@ function Get-Current-Branchname ()
   return $(git symbolic-ref --short -q HEAD)
 }
 
-function Get-Last-Version-Of-Branch-From-Tag-Exists ($Branchname)
+function Get-Last-Version-Of-Branch-From-Tag-Exists ($From, $To)
```

Comment:
Ah, yes, could have made a note. I just thought a separate ticket or PR would have been overkill.

![image](https://user-images.githubusercontent.com/40166342/108342531-32dc3180-71db-11eb-8f37-6989270c2a49.png)

Reply:
Would you kindly make a new slide with your change and write your name and change date at the top left like I started doing? I know it's a little dumb, but my initial proposal to remake the diagram with draw.io (or any other text-based diagram tool that can be checked into source control) was shot down because of budget reasons, so this "poor man's versioning" is better than nothing imo.

Reply:
I have inserted it as an extra slide and added an additional slide concerning project hotfixes.